corehq/apps/app_manager/models.py

```diff
@@ -3680,6 +3687,7 @@ class Application(ApplicationBase, TranslationMixin, HQMediaMixin):
                                           choices=app_strings.CHOICES.keys())
     commtrack_requisition_mode = StringProperty(choices=CT_REQUISITION_MODES)
     auto_gps_capture = BooleanProperty(default=False)
+    created_from = StringProperty()
```

Comment:
Could we call this `created_from_template_id` to make it more explicit that its a reference to a template?